utilities/blob_db/blob_db_impl.cc

```diff
@@ -0,0 +1,2073 @@
+      }
+
+      if (!blob_count) {
+        Log(InfoLogLevel::INFO_LEVEL, db_options_.info_log,
+          "0 blobs found in file %s", bfpath.c_str());
+        continue;
+      } else
+        Log(InfoLogLevel::INFO_LEVEL, db_options_.info_log,
+          "%" PRIu64 " blobs found in file %s", blob_count,
+          bfpath.c_str());
+
+      bfptr->setBlobCount(blob_count);
+      bfptr->setSNRange(sn_range);
+
+      if (bfptr->HasTimestamps()) {
+        bfptr->setTimeRange(ts_range);
+      }
+
+      if (bfptr->HasTTL()) {
+        ttl_range.second  = std::max(ttl_range.second,
+          ttl_range.first + (uint32_t)bdb_options_.ttl_range);
+        bfptr->setTTLRange(ttl_range);
+        std::time_t epoch_now = system_clock::to_time_t(system_clock::now());
+        if (ttl_range.second < epoch_now) {
+          Status fstatus = createWriter_locked(bfptr, true);
+          if (fstatus.ok())
+            fstatus = bfptr->writeFooterAndClose_locked();
+          if (!fstatus.ok()) {
+            // report error here
+            continue;
+          }
+        } else
+          open_blob_files_.insert(bfptr);
+      }
+    }
+
+    blob_files_.insert(std::make_pair(f_iter.first, bfptr));
+  }
+
+  return status;
+}
+
+////////////////////////////////////////////////////////////////////////////////
+//
+//
+////////////////////////////////////////////////////////////////////////////////
+Status BlobFile::readFooter_locked(blob_log::BlobLogFooter& bf) {
+
+  if (file_size_ < (blob_log::BlobLogHeader::kHeaderSize +
+    blob_log::BlobLogFooter::kFooterSize))
+    return Status::IOError("File does not have footer", PathName());
+
+  uint64_t footer_offset = file_size_ - blob_log::BlobLogFooter::kFooterSize;
+  // assume that ra_file_reader_ is valid before we enter this
+  assert(ra_file_reader_);
+
+  Slice result;
+  char scratch[blob_log::BlobLogFooter::kFooterSize+10];
+  Status s = ra_file_reader_->Read(footer_offset,
+        blob_log::BlobLogFooter::kFooterSize, &result, scratch);
+
+  if (!s.ok())
+    return s;
+
+  s = bf.DecodeFrom(&result);
+  return s;
+}
+
+////////////////////////////////////////////////////////////////////////////////
+// Should take a WriteLock on the mutex_
+//
+////////////////////////////////////////////////////////////////////////////////
+void BlobDBImpl::closeRandomAccess_locked(std::shared_ptr<BlobFile>& bfile) {
+  bfile->closeRandomAccess_locked();
+  open_file_count_--;
+}
+
+////////////////////////////////////////////////////////////////////////////////
+// Write mutex on file needs to be held
+//
+////////////////////////////////////////////////////////////////////////////////
+std::shared_ptr<RandomAccessFileReader> BlobDBImpl::openRandomAccess_locked(
+  std::shared_ptr<BlobFile> &bfile, Env *env, const EnvOptions& env_options) {
+  bool fresh_open = false;
+  auto rar = bfile->openRandomAccess_locked(env, env_options, &fresh_open);
+  if (fresh_open)
+    open_file_count_++;
```

Comment:
Reran lint and it pulled it into same line. I have tried to make sure that if there are
2 lines, I enclose it with { }